easybuild/easyconfigs/o/OpenFOAM-Extend/OpenFOAM-Extend-4.1-20210705_fix-private-PackedList.patch

```diff
@@ -0,0 +1,14 @@
+fix for:
```

Comment:
Is there something missing in this line?

Reply:
No, see next line